afs-ws/afs-ws-server-utils/src/main/java/com/powsybl/afs/ws/server/utils/AppDataBean.java

```diff
@@ -91,4 +92,27 @@ public void clean() {
             }
         }
     }
+
+    /**
+     * This method reinit only the computation manager (no effect on other connexions with other backends)
+     *
+     */
+    public void reinitComputationManager() {
+        try {
+            if (shortTimeExecutionComputationManager != null) {
+                shortTimeExecutionComputationManager.close();
+            }
+        } catch (Exception e) {
+            // Do nothing
+        }
+        try {
+            if (longTimeExecutionComputationManager != null) {
+                longTimeExecutionComputationManager.close();
+            }
+        } catch (Exception e) {
+            // Do nothing
+        }
+        shortTimeExecutionComputationManager = config.createShortTimeExecutionComputationManager();
+        longTimeExecutionComputationManager = config.createLongTimeExecutionComputationManager();
+    }
```

Comment:
should there not be a `new AppData` here too, like in the `init()`?

Reply:
No. Otherwise we would have used init() !
We wanted a method to reinit only the CALIN related connexion.